var/spack/repos/builtin/packages/py-pyomo/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyomo(PythonPackage):
+    """Pyomo is a Python-based open-source software package that supports a
+       diverse set of optimization capabilities for formulating and analyzing
+       optimization models."""
+
+    homepage = "https://github.com/Pyomo/pyomo"
+    url      = "https://github.com/Pyomo/pyomo/archive/5.6.6.tar.gz"
+
+    version('5.6.6', sha256='9330956b9fb244351ce76aaaf88688b5bdd03eebb122020cbee7b46e198a4110')
+
+    depends_on('py-appdirs', type=('build', 'run'))
```

Comment:
Missing a couple dependencies:
```python
depends_on('py-setuptools', type='build')
depends_on('py-cython', type='build')
```
The cython dep might not actually be required if they ship the C files pre-cythonized.
There's also some Python 2.6 deps:
```python
if sys.version_info < (2, 7):                                                   
    requires.append('argparse')                                                 
    requires.append('unittest2')                                                
    requires.append('ordereddict') 
```
You could also specify the Python version requirements:
```python
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*'
```

Reply:
Thanks, I missed those; fixed!
The python version should ensure python is >= 2.7 so I think the 2.6 dependencies can safely be left out.